### DISCUSSION TITLE
Help with SVG artwork

### DISCUSSION THREAD

**Discussion by PRProd** [PRProd/HA-Firemote#100] [General]
As new app shortcut requests come flooding in, I've found myself sometimes overwhelmed by trying to find or create the appropriate SVG artwork to go on each button.

To apply a brand logo to an app shortcut button requires two things:
 - A SVG file with a transparent background
 - A SMALL file size

I'm hoping that I can ask some of you to help me out with this task.  Is anyone interested?

**Comment by PRProd** [comment]
@ChristophCaina - Can I ask for your help here?

**Comment by ChristophCaina** [comment]
Hi, I will see how I can assist, but it might Take some time... Unfortunately, I am Not a Designer so I can also Just searching for Images ans might convert them...Also, it's quite busy here at the moment...Gesendet von Outlook für Android

**Reply by PRProd** [reply]
Thanks!  ...I'm not sure why I thought you were a designer, but somehow I had it in my mind that you are.  Sorry about that.

**Comment by PRProd** [comment]
## ~Cosmote TV Logo needed~
Official logo can be found here as a PNG: https://www.cosmotetvott.gr/assets/images/logo.png
![image](https://github.com/PRProd/HA-Firemote/assets/25486599/ac5e7db9-9fa6-4b45-8cad-727dd115587b)

This is the best I've done so far.  This SVG of 33.5K is way too big.  I need something smaller.
![tstLogo](https://github.com/PRProd/HA-Firemote/assets/25486599/00bceaa6-6ec6-4dab-9890-f4c1ad6a0d3f)

**Reply by PRProd** [reply]
This request has been completed.  I now have an SVG that is about half the size.  See the change here: https://github.com/PRProd/HA-Firemote/compare/v2.2.0...v2.2.1

**Comment by ChristophCaina** [comment]
No, I've Just used an online Tool to convert the Image the Last Time...I've created an Image, but I think it has a black Background and is also too big atm... Maybee, you can resize it or change the Background?Gesendet von Outlook für AndroidVon: Doug Nelson ***@***.***>Gesendet: Dienstag, 9. Mai 2023, 19:17An: PRProd/HA-Firemote ***@***.***>Cc: Christoph ***@***.***>; Mention ***@***.***>Betreff: Re: [PRProd/HA-Firemote] Help with SVG artwork (Discussion #100)
Cosmote TV Logo needed
Official logo can be found here as a PNG: https://www.cosmotetvott.gr/assets/images/logo.png

This is the best I've done so far.  This SVG of 33.5K is way too big.  I need something smaller.

**Comment by ChristophCaina** [comment]
![logo_adobe_express](https://github.com/PRProd/HA-Firemote/assets/26391061/05253ac7-1af7-4c10-869f-77546ca4bae1)

**Reply by PRProd** [reply]
Thanks for the effort!  After reducing the size (original is 163.4kB) .and removing the black background, the best I can do with that is 66.2kB, almost twice the size of the big one I already have.  :frowning_face: Bummer...
![237177317-05253ac7-1af7-4c10-869f-77546ca4bae1-min](https://github.com/PRProd/HA-Firemote/assets/25486599/d88ffb28-71e0-4e97-8dda-b888f68fb108)

**Comment by ChristophCaina** [comment]
And... The "TV" at the end got lost... :-(
Sorry, I don't really See how I could Support here :-(

**Reply by PRProd** [reply]
No worries @ChristophCaina!  I appreciate your willingness to help!  (also, the TV part might not be lost if you are viewing the SVG against a white background)
